dist/t/spec/features/0000_set_workers_spec.rb

```diff
@@ -0,0 +1,18 @@
+require "spec_helper"
+#for getting spec file
+require 'tmpdir'
+require "net/https"
+require "uri"
+
```

Comment:
I think we need those 3 `(tmpdir, net/https, uri)` only on **Preparation for building package obs-build** spec